packages/plugin-commands-installation/src/import.ts

```diff
@@ -114,18 +156,55 @@ export async function handler (
 async function readYarnLockFile (dir: string) {
   try {
     const yarnLockFile = await gfs.readFile(path.join(dir, 'yarn.lock'), 'utf8')
-    const lockJsonFile = await parseYarnLock(yarnLockFile)
-    if (lockJsonFile.type === 'success') {
-      return lockJsonFile.object
-    } else {
-      throw new PnpmError('GET_YARN_LOCKFILE_ERR', `Failed With ${lockJsonFile.type}`)
+    let lockJsonFile
+    const yarnLockFileType = getYarnLockfileType(yarnLockFile)
+    if (yarnLockFileType === YarnLockType.yarn) {
+      lockJsonFile = await parseYarnLock(yarnLockFile)
+      if (lockJsonFile.type === 'success') {
+        return lockJsonFile.object
+      } else {
+        throw new PnpmError('GET_YARN_LOCKFILE_ERR', `Failed With ${lockJsonFile.type}`)
```

Comment:
Don't put _ERR in the error code because each error code is automatically prepended with `ERR_PNPM_`